fetchQuotes.test.js

```diff
@@ -0,0 +1,12 @@
+import fetchQuotes from './fetchQuotes.js';
+
+describe('returns an object with a name, text, and image key value pair', () => {
+  it('should return a list of quotes in object format with a name, text, and image key value pair', async () => {
+    const quote = await fetchQuotes();
+    expect(quote).toEqual({
+      name: 'Bender',
```

Comment:
Remember you can use expect.any to handle random results - https://jestjs.io/docs/expect#expectanyconstructor.
Another option is to use mocks to return exact data - https://jestjs.io/docs/en/mock-functions#mocking-modules